tests/test_desmear.py

```diff
@@ -0,0 +1,62 @@
+import os
+import pytest
+import corgidrp
+import numpy as np
+import corgidrp.detector as detector
+from astropy.time import Time
+from corgidrp.l2a_to_l2b import desmear
+from corgidrp.mocks import create_default_headers
+from corgidrp.data import Image, Dataset, BadPixelMap
+
+old_err_tracking = corgidrp.track_individual_errors
+
+def test_desmear():
+    # Tolerance for comparisons
+    tol = 1e-12
+
+    corgidrp.track_individual_errors = True
```

Comment:
same comment as on the other PR. I think it's more important to test when this is False, since that's the standard use case.

Reply:
Run tests succesfully